lib/data/models/browser_bookmark_item.dart

```diff
@@ -28,6 +28,6 @@ class BrowserBookmarkItem with _$BrowserBookmarkItem {
         title: title,
         url: url,
         faviconUrl: faviconUrl,
-        sortingOrder: DateTime.now().millisecondsSinceEpoch,
+        sortingOrder: DateTime.now().millisecondsSinceEpoch.toDouble(),
```

Comment:
What the problem with int?

Reply:
It's just a cheap sort pattern: we can put item between two other by calculating sorting order index as `(next + prev) / 2 + prev` without rewriting other items.